CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [7.0.0]
+### Changed
+- **BREAKING:** Bump minimum Node version to 16 ([#139](https://github.com/MetaMask/json-rpc-engine/pull/139))
```

Comment:
Maybe this could be rephrased; there was no minimum Node.js version prior to this release

Reply:
Good point!